base/shell/explorer/traywnd.cpp

```diff
@@ -2605,6 +2607,8 @@ class CTrayWindow :
     // We have to draw non-client area because the 'Show Desktop' button is beyond client area.
     void DrawShowDesktopButton()
     {
+        if (!m_ShowDesktopButton.m_hWnd)
```

Comment:
I'd use `!m_ShowDesktopButton.IsWindow()`.
That also covers the case where the window is destroyed, but the m_hWnd is not cleaned up yet.
(Same with `PtInButton`)

Reply:
Done.